Makefile.am

```diff
@@ -471,20 +475,34 @@ endif
 libcollectdclient_la_SOURCES = \
 	src/libcollectdclient/client.c \
 	src/libcollectdclient/network.c \
-	src/libcollectdclient/network_buffer.c
+	src/libcollectdclient/network_buffer.c \
+	src/libcollectdclient/network_parse.c \
+	src/libcollectdclient/server.c
 libcollectdclient_la_CPPFLAGS = \
 	$(AM_CPPFLAGS) \
-	-I$(srcdir)/src/libcollectdclient/collectd \
-	-I$(top_builddir)/src/libcollectdclient/collectd \
+	-I$(srcdir)/src/libcollectdclient \
+	-I$(top_builddir)/src/libcollectdclient \
 	-I$(srcdir)/src/daemon
-libcollectdclient_la_LDFLAGS = -version-info 1:0:0
-libcollectdclient_la_LIBADD = 
+libcollectdclient_la_LDFLAGS = -version-info 2:0:1
+libcollectdclient_la_LIBADD = -lm
```

Comment:
we should do some detection as for when we need to link with libmath, but let's do that later. What function / symbol needs this?